assistant/src/main/java/com/ibm/watson/assistant/v1/Assistant.java

```diff
@@ -2248,6 +2248,12 @@ public void setVersion(final String version) {
           com.ibm.cloud.sdk.core.util.GsonSingleton.getGson()
               .toJsonTree(updateDialogNodeOptions.newNextStep()));
     }
+    if (updateDialogNodeOptions.deleteNextStep() != null && updateDialogNodeOptions.deleteNextStep()) {
+      contentJson.add(
+              "next_step",
+              com.ibm.cloud.sdk.core.util.GsonSingleton.getGson()
+                      .toJsonTree(null));
```

Comment:
Does this actually put `null` into `next_step` in the json? want to make sure gson isn't disregarding null values

Reply:
yes, this is the output from `nc -l` 
```
{"dialog_node":"Test2d28f9c6e-8864-4bbb-8ab8-81d18b740ce9","description":"Updated description of Test79155c70-05aa-47a3-b1e2-5557109f89b3","next_step":null}
```

Reply:
It does. We looked at the final request sent to the service and it `inserts` null into the request